diff --git a/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts b/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts
index df81ae7..c68ecd8 100644
--- a/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts
+++ b/node_modules/@react-three/fiber/dist/declarations/src/three-types.d.ts
@@ -47,12 +47,6 @@ export interface ThreeElements extends Omit<ThreeElementsImpl, 'audio' | 'source
     threeLine: ThreeElementsImpl['line'];
     threePath: ThreeElementsImpl['path'];
 }
-declare module 'react' {
-    namespace JSX {
-        interface IntrinsicElements extends ThreeElements {
-        }
-    }
-}
 declare module 'react/jsx-runtime' {
     namespace JSX {
         interface IntrinsicElements extends ThreeElements {
